products/sle/main.pm

```diff
@@ -922,6 +922,26 @@ sub load_feature_tests() {
     loadtest "feature/feature_console/suseconnect.pm";
 }
 
+sub load_online_migration_tests() {
+    # stop packagekit service and more
+    loadtest "online_migration/sle12_online_migration/online_migration_setup.pm";
+    loadtest "online_migration/sle12_online_migration/register_system.pm";
```

Comment:
isn't that updated SUSEConnect provided as a maintenance update to sle12 GA?

We can have whatever scenarios you want, for offline migration we have combinations of minimal installs, full installs, both with and without the sdk and with the sdk, geo, and ha.